### PR TITLE
feat: puppeteer page options

### LINKED ISSUE #1: Feature: Allow puppeteer options to specify page formatting
Hi @Igloczek, 

Thanks for the plugin! 

While using it I realized I need some page formatting options. And Puppeteer's `page.pdf()` function takes in several parameters as specified here: https://github.com/puppeteer/puppeteer/blob/v2.1.1/docs/api.md#pagepdfoptions. It would be very helpful to pass those options from the plugins in vuepress to provide some custom page formatting operations, as the default is somewhat very basic.

### PR DESCRIPTION
**Pull request by aporan**
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +38,18 @@ To run this plugin on Gitlab CI you may want to run Chrome with `no-sandbox` fla
 module.exports = {
   plugins: [
     ['@snowdog/vuepress-plugin-pdf-export', {
+      // to pass puppeteer launch options
       puppeteerLaunchOptions: {
         args: ['--no-sandbox', '--disable-setuid-sandbox']
+      },
+      // to pass puppeteer page options
```
Comment (Igloczek) [inline README.md]: Everything is alright except this little thing. This part is related only to Gitlab CI, it's not a generic example of how to pass some data.

Comment (aporan) [reply README.md]: okay, let me remove that.

Comment (aporan) [reply README.md]: done, and resolving this

Comment (aporan) [reply README.md]: hi! if you could merge it in and make a release, it'd be very helpful

Comment (Igloczek) [reply README.md]: I'll try to handle that tomorrow

Comment (aporan) [reply README.md]: Ah, thank you very much ! It'll help me in my own Gitlab pipeline

Comment (aporan) [reply README.md]: hiya @Igloczek, just doing a friendly bump

**Review by Igloczek — APPROVED** [review]

**Comment by Igloczek** [comment]
Sorry for the long waiting! 
You can now get those changes in version 1.1.0, thanks for the contribution 🙇

**Comment by aporan** [comment]
thanks a bunch man! I can now use this in our own gitlab pipeline! Enjoy your weekend :)

**Pull request merged**